targeteditor/src/main/scala/explore/targeteditor/MagnitudeForm.scala

```diff
@@ -174,16 +202,21 @@ object MagnitudeForm {
             .setInitialStateFull(tableState)
 
           React.Fragment(
-            <.div(<.label("Magnitudes")),
-            Segment(
-              tableMaker.makeTable(
-                options = options,
-                data = props.magnitudes.toListOfViews(_.band).toJSArray,
-                headerCellFn = None,
-                tableClass = Css("ui very compact table"),
-                footer = footer
+            <.div(
+              <.label("Magnitudes"),
+              Segment(attached = SegmentAttached.Attached,
+                      compact = true,
+                      clazz = ExploreStyles.MagnitudesTableContainer
+              )(
+                tableMaker.makeTable(
+                  options = options,
+                  data = props.magnitudes.toListOfViews(_.band).toJSArray,
+                  headerCellFn = Some(c => <.th(Column.all.get(c.id.toString).foldMap(_.label))),
```

Comment:
If you use `.setHeader` on the column definitions above you can should be able to just use `TableMaker.basicHeaderCellFn()`.  That will also apply any properties necessary for react-table to the header cells. In this case it is only 'colspan` and `role`, but in other cases they can be significant.

I think this would also eliminate the need for the Column list, too.